libs/sdk-ui-kit/src/Icon/icons/DrillToDashboard.tsx

```diff
@@ -0,0 +1,21 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+
+import { IIconProps } from "../typings";
+
+/**
+ * @internal
+ */
+export const DrillToDashboard: React.FC<IIconProps> = ({ color }) => {
+    return (
+        <svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg">
+            <g id="Artboard" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+                <polygon
+                    id=""
```

Comment:
What is this character? Do we even need those externally generated attributes in icons?

Reply:
I guess we don't. I already made a bit of cleanup within SVGs and was worried to not mess with its look. To answer your question, this character is just a copy paste from the original.